docker/README.md

```diff
@@ -16,13 +16,19 @@ Supported architectures are:
 - `armv7` (Ex. Raspberry PI)
 - `arm64` (Ex. OrangePI NanoPI)
 
+Supported tags are:
+- `latest` for the newest official release.
+- `master` most recent fixes and merged PRs.
```

Comment:
There are also `sha-<commit-sha>` and `<version>` tags (like `2.1.0`)

Reply:
Ok, will add that after work. How often are you planning to cut the master tag? I could add that too.

Reply:
> How often are you planning to cut the master tag
What do you mean?

Reply:
If I’m not mistaken, with zwavejs/zwavejs2mqtt:latest the image gets built when there is a release, so for the master tag, does it get built daily? Just thinking to highlight the benefits of it.

Reply:
It's built on every push to `master` branch

Reply:
Please check again when you get a chance. I think it should be good now.